### PR TITLE
[New Tool] Kleborate

### PR DESCRIPTION
**Pull request by shiltemann**
Adds a wrapper for the [Kleborate](https://github.com/katholt/Kleborate) tool

There was a wrapper for this by toolshed user "dazeone" which I used as a starting point. If anybody knows this user please let me know so I can credit them for the first commit.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/kleborate/kleborate.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        <data name="kaptive_k_outfile" format="tabular" label="${tool.name} on ${on_string}: K-loci screening results" >
+            <filter>kaptive_k</filter>
+        </data>
+        <data name="kaptive_o_outfile" format="tabular" label="${tool.name} on ${on_string}: O-loci screening results" >
+            <filter>kaptive_o</filter>
+        </data>
+    </outputs>
+
+    <tests>
+        <test><!-- test with resistance, kaptive k and kaptive o outputs -->
```
Comment (bgruening) [inline tools/kleborate/kleborate.xml:78]: can you test for the expected_num_outputs here please?

File: tools/kleborate/kleborate.xml [hunk 2]
```diff
@@ -0,0 +1,182 @@
+<tool id="kleborate" name="Kleborate" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>screen genome assemblies of Klebsiella pneumoniae</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2.2.0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bgruening) [inline tools/kleborate/kleborate.xml:5]: should that be a 0?

**Review by bgruening — APPROVED** [review]

File: tools/kleborate/kleborate.xml (hunk 2 above)
Comment (shiltemann) [reply tools/kleborate/kleborate.xml]: :+1:

**Comment by bgruening** [comment]
Merci!

**Pull request merged**